examples/use_as_lib.py

```diff
@@ -25,7 +26,8 @@ def task_404(self):
 env.create_local_runner()
 
 # start a WebUI instance
-env.create_web_ui("127.0.0.1", 8089)
+options = parse_options()  # TODO: Confirm this is accurate
+env.create_web_ui(options, "127.0.0.1", 8089)
```

Comment:
Cant we make this a non-breaking change by just putting the parameter last and defaulting to use parse_options() inside in create_web_ui() if nothing is provided?

Reply:
Sounds good to me, I'll give it a shot

Reply:
After reading your other comment, I realized that my approach here was probably very unnecessary. I'm already passing in `show_locustfiles`, which I can use for the same check in `/swarm` to see if the locustfile in the payload needs to be loaded.